docs: add downloadApiSpec contextual menu option (#6024)
mintlify/docs: ai/contextual-menu.mdx

| **Connect to Cursor** | `cursor` | Installs your hosted MCP server in Cursor |
| **Connect to VS Code** | `vscode` | Installs your hosted MCP server in VS Code |
| **Connect to Devin** | `devin-mcp` | Installs your hosted MCP server in Devin |

> | **Download API spec** | `downloadApiSpec` | Downloads your deployment's OpenAPI spec. If multiple specs are configured, downloads them as a zip archive. Only appears on API reference pages. |

Reviewer: ```suggestion
| **Download API spec** | `download-spec` | Downloads your deployment's OpenAPI spec. If there are multiple specs, downloads them as a zip archive. Only appears on API reference pages. |
```

  />
</Frame>

<Note>
  The `downloadApiSpec` option only appears on API reference pages (pages whose frontmatter declares `api`, `openapi`, or `asyncapi`). On deployments with authentication or user authentication enabled, only authenticated viewers can download the spec.
</Note>

> 

Reviewer: ```suggestion
```

      "mcp",
      "cursor",
      "vscode",
      "devin-mcp",

>       "downloadApiSpec"

Reviewer: ```suggestion
      "download-spec"
```
